event-service-dto/src/main/java/com/restapi/EventRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.restapi;
+
+import com.restapi.dto.Event;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface EventRepository extends JpaRepository<Event, Integer> {
+
+    @Query("SELECT e FROM Event e WHERE title = :title")
```

Comment:
i assume that this method will work even without the explicit query, probably you just need to rename the method
https://www.baeldung.com/spring-data-derived-queries

Reply:
> i assume that this method will work even without the explicit query, probably you just need to rename the method https://www.baeldung.com/spring-data-derived-queries

changed

Reply:
> i assume that this method will work even without the explicit query, probably you just need to rename the method https://www.baeldung.com/spring-data-derived-queries

changed